src/gluonts/distribution/distribution.py

```diff
@@ -48,6 +49,16 @@ class Distribution:
     arg_names: Tuple
     is_reparameterizable = False
 
+    _dtype: DType = np.float32
```

Comment:
Is this property set by all subtypes of `Distribution` upon construction?

Reply:
Yeah, its a class variable. I coded this analogous to DistributionOutput.

Reply:
I don’t think this is ideal (neither here nor in DistributionOutput, don’t get me wrong): I think dtype should better be a property of the object, either explicitly set upon construction, or inferred by the dtype of the constructor arguments.